### PR TITLE
feat: Support array value type

### LINKED ISSUE #112: Support array data types
Device SDK and Core services support array data type in Geneva.  Virtual Device Service should also support them for testing.

### PR DESCRIPTION
**Pull request by FelixTing**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

device-virtual-go cannot generate random values for array value types.
Issue Number: fix #112 

## What is the new behavior?
device-virtual-go can now generate random values ​​for the following array value types:
- BoolArray
- Int8Array, Int16Array, Int32Array, Int64Array
- Uint8Array, Uint16Array, Uint32Array, Uint64Array
- Float32Array, Float64Array

The array size is hardcoded as 5.

Added device resources with these array value types to the sample device profiles, the related device commands and core commands are added as well.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

File: internal/driver/helper.go [hunk 1]
```diff
@@ -0,0 +1,174 @@
+package driver
```
Comment (cloudxxx8) [inline internal/driver/helper.go:1]: add license string to any new file

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Comment by cloudxxx8** [comment]
please rebase

**Comment by FelixTing** [comment]
rebased

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**